.travis.yml

```diff
@@ -106,6 +106,7 @@ stage_osx: &stage_osx
 
 # Define the order of the stages.
 stages:
+  - wheels
```

Comment:
Should this stage be called `deploy` and come last in the chain?

Reply:
I did it first because it only gets triggered on tag events. We've already run the tests at least once (for the PR) and more realistically twice (for the merge event) for the state of master at that point. So having the travis build wait for the tests to run again was an unnecessary delay especially when debugging any failures during a release where we want the fastest turnaround. Ideally I would skip the test and compile stages on tag events (that's what I did on azure for windows) but I didn't know if there was a .travis.yml syntax for conditions on stages off the top of my head.

As for the name of the stage I'm fine with renaming it if you feel strongly about it.

Reply:
I see, I think you can skip the other stages then by having `if: tag IS blank AND (<any other conds>)` according to [travis docs](https://docs.travis-ci.com/user/conditions-v1).

Reply:
I would still call it deploy though if its deploying the wheels ( `deploy` seems to be the standard tag used in the Travis CI documentation)

Reply:
ok, updated the log and renamed it to deploy in: https://github.com/Qiskit/qiskit-aer/pull/168/commits/f8b75c1d2acf63cee7e82eea8da95ba3b36507e7